.gitignore

```diff
@@ -1,5 +1,7 @@
 *.o
 
+xapian-applications/omega/omindex_*
+
```

Comment:
The idea is to use similar names in next libraries

Reply:
OK, but really this would be better put in `xapian-applications/omega/.gitignore` - otherwise the ignore rules specific to that directory are split up.

Reply:
My mistake, I'm going to change it.